proposal.h

```diff
@@ -0,0 +1,14 @@
+/* one */
+proto:
+int nrf_setdnsaddr(const char *dns, size_t len);
+usage:
+nrf_setdnsaddr("8.8.8.8", strlen("8.8.8.8)"));
+
+/* two */
+proto:
+int nrf_setdnsaddr(int family, void *in_addr);
+usage:
+struct nrf_in_addr addr;
+nrf_inet_pton(AF_INET, "8.8.8.8", &addr);
```

Comment:
so if we have to use `nrf_inet_pton` anyway, I don't see a point of doing this outside of the function. Theoretically, we could use `net_pton` and cast from in_addr/in6_addr to nrf_in_addr/nrf_in6_addr (these are binary compatible), but I'm not convinced it's a neat solution. What would you say?